ElfUtils/LinuxMap.cpp

```diff
@@ -80,35 +108,15 @@ ErrorMessageOr<std::vector<ModuleInfo>> ParseMaps(std::string_view proc_maps_dat
   for (const auto& [module_path, address_range] : address_map) {
     // Filter out entries which are not executable
     if (!address_range.is_executable) continue;
-    if (!std::filesystem::exists(module_path)) continue;
-    ErrorMessageOr<uint64_t> file_size = FileSize(module_path);
-    if (!file_size) continue;
-
-    ErrorMessageOr<std::unique_ptr<ElfFile>> elf_file = ElfFile::Create(module_path);
-    if (!elf_file) {
-      // TODO: Shouldn't this result in ErrorMessage?
-      ERROR("Unable to load module \"%s\": %s - will ignore.", module_path,
-            elf_file.error().message());
-      continue;
-    }
 
-    ErrorMessageOr<uint64_t> load_bias = elf_file.value()->GetLoadBias();
-    // Every loadable module contains a load bias.
-    if (!load_bias) {
-      ERROR("No load bias found for module %s", module_path.c_str());
+    auto module_info =
```

Comment:
I generally prefer seeing the type rather than `auto`: for example just from the name it's unclear that `CreateModule` returns `ErrorMessageOr`.

Reply:
done.